src/psyclone/domain/lfric/arg_ordering.py

```diff
@@ -423,6 +432,21 @@ def stencil_unknown_extent(self, arg, var_accesses=None):
 
         '''
 
+    @abc.abstractmethod
+    def stencil_2d_unknown_extent(self, arg, var_accesses=None):
+        '''Add stencil information to the argument list associated with the
```

Comment:
Please extend this docstring to make it clear how this differs from `stencil_unknown_extent`.

Reply:
Added identifier to distinguish from stencil_unknown_extent